docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -474,9 +474,18 @@ Users can instruct the container to not remove any of their specific beans (even
  
 Furthermore, extensions can eliminate possible false positives by producing `UnremovableBeanBuildItem`.
 
+The container will also remove bean producing methods (methods annotated with `@Produces`) from producer beans if the method produces an unused bean.
+You can also annotate bean producing methods with `io.quarkus.arc.Unremovable`.
+
 Finally, Quarkus provides a middle ground for the bean removal optimization where application beans are never removed whether or not they are unused,
 while the optimization proceeds normally for non application classes. To use this mode, set `quarkus.arc.remove-unused-beans` to `fwk` or `framework`.
 
+If you feel that one of your beans, or one of your methods are removed,
+you can check it by enabling Arc producer beans logging via the following line in your application.properties.
```

Comment:
Let's reword this to:
`In order to see more information about which beans are being removed, you can enable additional logging via the following line in your application.properties.`

Reply:
done